### DISCUSSION TITLE
Instead of making enemies faster, make the duck harder to control?

### DISCUSSION THREAD

**Discussion by NNathaneel** [Quit975/DuckyGame#55] [Ideas]
I thought of this as an effect the ice surface would make? 
The duck would be sliding in a direction determined by the lastly pressed key, which would take more skill to properly navigate the duck. 
The effect would take place on X frogs collected or would gradually increase.

This is probably for later as a suggestion on how to make the game harder, until our fight with Pepe the Boss 😅

**Comment by Quit975** [comment]
I like that idea! Maybe at the beginning of the game a 'biome' where the game takes place would be selected (randomly or by the player) and depending on which one was chosen, there would be different negative effects after catching the - so in the ice world the effect could be the one you described.

However, since this is a gameplay /design idea, converting it to a discussion for now
